uint8 UPDATE_POSE=0
uint8 UPDATE_MARKER=1
uint8 CONTINUOUS_POSE_ON=2
uint8 CONTINUOUS_POSE_OFF=3
uint8 command